### PR TITLE
Update/start9 offerings

### PR DESCRIPTION
**Pull request by gStart9**

### CODE REVIEW

File: node/compare.md [hunk 1]
```diff
@@ -16,22 +16,21 @@ tags: [books, test]
...
+| [Nodl Dojo](/node/pap/#nodl-dojo)             | RockPi 4                         | Electrs                | LND       | Whirlpool      | Yes            | No             | Yes  |
+| [RoninDojo Tanto](/node/pap/#ronindojo-tanto) | RockPro 64                       | Electrs                | None      | Whirlpool      | Yes            | Yes            | Yes  |
+| [RoninDojo](h/node/diy/#ronindojo)            | Odroid N2, RockPro 64 and RPi    | Electrs                | None      | Whirlpool      | Yes            | Yes            | Yes  |
+| [RaspiBlitz](/node/pap/#raspiblitz)           | RPi 4                            | Electrs                | LND       | JoinMarket     | Yes            | Yes            | No   |
+| [RaspiBlitz DIY](/node/diy/#raspiblitz)       | RPi 3 and RPi 4                  | Electrs                | LND       | JoinMarket     | Yes            | Yes            | No   |
+| [Start9](/node/pap/#start9-server)            | RPi 4, x86_64, VM                | Electrs                | LND+CLN   | JoinMarket     | Yes            | Yes            | No   |
```
Comment (gStart9) [inline node/compare.md:32]: This is the only line that changed, everything else was table alignment (by 'Prettier')

File: node/diy.md [hunk 2]
```diff
@@ -261,6 +262,53 @@ The RoninDojo team have provided their own recommended [shopping list](https://w
 
 ***
 
+## Start9
```
Comment (gStart9) [inline node/diy.md:265]: Added Start9's DIY details to the DIY page

File: node/pap.md [hunk 3]
```diff
@@ -23,35 +23,37 @@ Plug and Play node boxes offer a convenient way to run your own node. Most run o
...
+|                                             | $419 (SSD)                        |
+| [Nodl One](#nodl-one)                       | $499                              |
+|                                             | +$50 w/ kill switch               |
+|                                             | +$50 w/ Dojo support              |
+| [Nodl Dojo](#nodl-dojo)                     | $849                              |
+| [RaspiBlitz](#raspiblitz)                   | 269€ (2GB)                        |
+|                                             | 299€ (4GB)                        |
+|                                             | 369€ (8GB)                        |
+|                                             | 379€ (4GB) w/ terminal style case |
+| [Start9](#start9-server)                    | $199 Lite (8GB Pi)                |
```
Comment (gStart9) [inline node/pap.md:51]: These 3 offerings are the only thing that actually changed in this table, everything else is alignment (by 'Prettier')

**Comment by BitcoinQnA** [comment]
Thank you!

**Pull request merged**